SPECS/python-virtualenv/python-virtualenv.spec

```diff
@@ -19,13 +20,11 @@ BuildRequires:  python3-devel
 BuildRequires:  python3-setuptools_scm
 BuildRequires:  python3-xml
 BuildRequires:  python3-wheel
-
-%if 0%{?with_check}
 BuildRequires:  python3-pip
```

Comment:
Is pip really required for the build when not running tests?

Reply:
Double checked and is not. Thanks :)